druid/setup.py

```diff
@@ -27,7 +27,7 @@ def get_dependencies():
         return f.readlines()
 
 
-CHECKS_BASE_REQ = 'datadog-checks-base>=4.2.0'
+CHECKS_BASE_REQ = 'datadog-checks-base>=11.0.0'
```

Comment:
We probably had this discussion before, but should this prompt us to make changes like these `changelog/Added` at least? Means that it can't be installed on older Agents anymore.

Reply:
yup, I didnt add any changelog label, my bad